src/styles/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+  width: 96px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+
+    &:nth-child(-n + 4) {
+      background-image: url(/src/images/star-active.svg);
+    }
```

Comment:
The logic of your stars styling is incorrect. The number of active stars should not be hardcoded into the CSS. Instead, you should create separate classes for all possible star ratings (from 1 to 5) and then add the appropriate class to the stars block in your HTML. This way, you can control the number of active stars by changing the class in the HTML, not the CSS.